docs/docs/references/users/README.md

```diff
@@ -214,3 +214,69 @@ Updating a user's _custom data_ will overwrite (not merge) its original content
 :::
 
 ### Identities
+
+_Identities_ contain the user info retrieved from social sign-in
+(a.k.a. sign-in with a social connector).
+
+The user info from individual social identity providers varies by provider,
+but it will typically include the following:
```

Comment:
Sounds strange

Reply:
OK to me

Reply:
According to another doc
<img width="236" alt="image" src="https://user-images.githubusercontent.com/10594507/176816787-0e093f7d-aa15-4369-91d3-d8a79d1f5f33.png">

update to?

```
and it typically includes the following:
```